MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -27,7 +28,9 @@ public class ProbeResultNormalizer
 
         private readonly char[] _nameDelimiters = { '/', '|', ';', '\\' };
 
-        private readonly CultureInfo _usCulture = new CultureInfo("en-US");
+        private readonly Regex _performerPattern = new (@"(?<name>.*) \((?<instrument>.*)\)");
```

Comment:
Now you will create a new `Regex` instance for each ffprobe process, you can make this static instead (it's thread safe)